.zuul.yaml

```diff
@@ -1,16 +1,26 @@
 ---
+
+- job:
+    name: openstack-project-manager-mypy
+    parent: mypy
+    pre-run: playbooks/pre.yml
+    timeout: 10800
```

Comment:
Nit: the timeout seems to be excessive, omitting this setting and running with the default timeout should be fine